apps/storefront/app/komponenter/layout.tsx

```diff
@@ -19,7 +19,7 @@ const Layout = ({ children }: { children: React.ReactNode }) => {
     <main id='main' className={classes.page}>
       <Banner color='blue'>
         <BannerIcon>
-          <ComponentFillIcon fontSize={'2.5rem'} />
+          <ComponentFillIcon fontSize={'2.5rem'} aria-hidden='true' />
```

Comment:
Is this needed now that you add the Slot and `aria-hidden` in `BannerIcon`?

Reply:
Not sure, but it at least ensures it is hidden 🤷🏻‍♂️